packaging/crio.conf.d/microshift-ovn.conf

```diff
@@ -2,8 +2,3 @@
 # ovn-kubernetes is the name configured by OVN-K in /etc/cni/net.d/ config file
 # by declaring this CRI-O will wait until that network is configured.
 cni_default_network = "ovn-kubernetes"
-
```

Comment:
I think these settings were a key part of managing the size of the CNI implementation, so I'm worried that moving them to a separate file is going to cause problems when the new optional RPM is not installed.

cc @zshi-redhat @mangelajo

Reply:
I see. The only thing we need to clarify is if the desire is that the default behavior is single CPUSet for microshift as a whole. If that's the case then I think encapsulating that configuration in it's own file seems easier to maintain. Since CRIO will merge these values and the last file to be read in updates the value then the workload file will always override this value. However, if the optional CNI config has it's own limits, then it makes perfect sense for it to be delivered with those limits, either way though, there should be no issue with adding this back in.

Reply:
OK, let's see what Zenghui and Miguel think when they have a chance to take a look.

Reply:
It makes sense to move the workload crio settings to its own file. ovnk can still use it w/o any changes (the annotation was already added in ovnk daemonset), other microshift pods can benefit from the settings as well.

Reply:
Regarding the default behavior to pin a single cpu, I think it is reasonable in most cases since our target is small form factor device. It may become useful to make the cpuset configurable if the target changes in the future.